packages/docs/static/themes/core-theme.css

```diff
@@ -529,4 +375,158 @@
 --vertical-nav-label-icon__color: #262626;
 --vertical-nav-label__border-color--current: #0071bc;
 --vertical-nav-label__color--current: #0071bc;
-}
+--animationease-in-out-expo: cubic-bezier(1, 0, 0, 1);
+--animationspeed-base: 1;
+--animationspeed-1: 0.25s;
+--animationspeed-2: 0.3s;
+--animationspeed-3: 0.5s;
+--animationspeed-4: 0.8s;
+--colorwhite: #ffffff;
+--colorblack: #000000;
+--colortransparent: #ffffff00;
+--colortransparent-black-alpha50: #00000080;
+--colortransparent-black-alpha25: #00000040;
+--colortransparent-white-alpha50: #ffffff80;
```

Comment:
It took me running the doc site to notice this. There is no delimiter between `color` and the color name! (And same with `animation` and `speed`

Reply:
doh! i think it's cuz i wrote the css vars to the output before the !file.baseName.includes('component') logic. checking that theory now.